src/alert-unified2-alert.c

```diff
@@ -254,14 +254,11 @@ void TmModuleUnified2AlertRegister(void)
  *  \param t Thread Variable containing  input/output queue, cpu affinity etc.
  *  \param aun Unified2 thread variable.
  */
-
 int Unified2AlertCloseFile(ThreadVars *t, Unified2AlertThread *aun)
 {
-    if (aun->unified2alert_ctx->file_ctx->fp != NULL) {
```

Comment:
I don't want these 'fixes' that are removing braces. Keeping the braces makes the code more robust against later insertion errors.